modules/book_counter.js

```diff
@@ -0,0 +1,19 @@
+// Show a counter next to link1 after adding a book
+export function showBookCounter(
```

Comment:
- [ ] Great job creating your app, it works perfectly! However, as this is an ES6 activity, it would be better to use arrow functions, as this is a better practice in ES6, please do this for all the functions.